cloudbridge/cloud/base/resources.py

```diff
@@ -782,6 +782,10 @@ def to_json(self):
         js['firewall'] = self.firewall.id
         return js
 
+    @property
+    def parent(self):
```

Comment:
@baizhang @chiniforooshan @mbookman Should the parent property be moved to the GCEFirewall class? It doesn't seem to apply to other providers?